fem/coefficient.cpp

```diff
@@ -485,32 +485,32 @@ VectorSumCoefficient::VectorSumCoefficient(int dim)
 }
 
 VectorSumCoefficient::VectorSumCoefficient(VectorCoefficient &_A,
-                                           VectorCoefficient &_B,
+                                           VectorCoefficient &B_,
```

Comment:
Do we want to rename the rest of the _* variables in these files to avoid trouble like this int he future?